src/generate-query.ts

```diff
@@ -586,9 +593,30 @@ function getArgsAndVars(
     const argType = schema.getType(getTypeName(arg.type))
     if (typeof variableValues[varName] === 'undefined') {
       if (isEnumType(argType)) {
+        // Enum values can be randomly selected
         variableValues[varName] = getRandomEnum(argType)
       } else if (config.providePlaceholders) {
-        variableValues[varName] = getDefaultArgValue(arg.type)
+        variableValues[varName] = getDefaultArgValue(schema, config, arg.type)
+
+        // Add default values for unfulfilled listSize directive if applicable
+        const listSizeDirective = getListSizeDirective(field)
+        if (listSizeDirective) {
+          const value = getListSizeDirectiveDefaultValue(
```

Comment:
I think this doesn't work as intended with `requireOneSlicingArg`. With the current way that `getListSizeDirectiveDefaultValue` is being called, it doesn't matter whether `requireOneSlicingArg` is set to true or not, we can only add at most one slicing argument. What do you think?

Reply:
Let me rephrase, it works with multiple input object types but if there are multiple integer slicing args, this doesn't work right

Reply:
yea I agree, I think this is a similar disconnect to the comment I made here
>hmmm... yea I guess there is a disconnect between the slicingargs list (could contain more than one argument root) and the caller which just wants a value for one argument
https://github.com/IBM/graphql-query-generator/pull/38/files#r650224767